pressSpace4me.m

```diff
@@ -1,19 +1,17 @@
 function pressSpace4me
```

Comment:
```suggestion
function pressSpace4Me
```

Reply:
actually should be `pressSpaceForMe` in line with `degToPix`, though I like the numbers feel so yo

Reply:
Good point.
Will make a change to both.

Reply:
If you find a way to change a var name across files in a path do let me know!